src/frame/endpoints/GroupApiEndpoints.ts

```diff
@@ -63,14 +64,13 @@ function groupGet(groupID: string) {
 function groupCreate(groupID: string, createPayload: GroupCreatePayload) {
     const userPublicKeyString = publicKeyToBase64(createPayload.userPublicKey);
     let memberList;
-    if (createPayload.transformKey) {
-        memberList = [
-            {
-                userId: createPayload.userID,
-                userMasterPublicKey: userPublicKeyString,
-                transformKey: transformKeyToBase64(createPayload.transformKey),
-            },
-        ];
+
+    if (createPayload.transformKeyGrantList) {
```

Comment:
When you make the change to have `transformKeyGrantList` not optional we should make this more clear what we're checking here, change this to 
```suggestion
    if (createPayload.transformKeyGrantList.length > 0) {
```